src/pages/ProductPage.jsx

```diff
@@ -0,0 +1,23 @@
+import { Flex } from "@sparrowengg/twigs-react";
+import ProductDetailView from "../components/ProductDetailView";
+import SideBar from "../components/SideBar";
+import BagItems from "../components/BagItems";
+const ProductPage = () => {
+  return (
+    <>
+      <Flex css={{ height: "100%" }} gap="$5">
+        <Flex>
+          <SideBar />
+        </Flex>
```

Comment:
@arun-121 can you remove unused flexes here

Reply:
removed the same